index.html

```diff
@@ -45,7 +45,7 @@ <h1 class="primary-text">I'm Oscar <br> Glad to see you!</h1>
           you see and have a project you need coded, don't hestiate to contact
           me.
         </p>
-        <h3 class="letsconnect">LET'S CONNECT</h3>
+        <h2 class="letsconnect">LET'S CONNECT</h2>
 
         <ul class="socialmedia-list">
```

Comment:
- [ ] Good job adding alt attributes to your social icons 👍 . I think you should consider giving the alt attributes a more descriptive and appropriate alternative text, that's useful for interacting with or understanding the page to enable screen readers to navigate your website easily. e.g ;

instead of: **`<li class="socials"><img src="images/angelist.png" alt="ange" /></li>`**

you can write `<li class="socials"><img src="images/angelist.png" alt="angellist" /></li>`